### ISSUE TITLE
Instruction unclear that conda is absolutely required 

### ISSUE THREAD

**Issue by dany-nonstop** [pytorch/benchmark#1533]
Error encountered while installing from python venv. It seems conda is required absolutely.

```
running setup for /home/user/try/benchmark/torchbenchmark/models/doctr_det_predictor...FAIL
Error for /home/user/try/benchmark/torchbenchmark/models/doctr_det_predictor:
---------------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/user/try/benchmark/torchbenchmark/models/doctr_det_predictor/install.py", line 12, in <module>
    pip_install_requirements()
  File "/home/user/try/benchmark/torchbenchmark/models/doctr_det_predictor/install.py", line 8, in pip_install_requirements
    subprocess.check_call(["conda", "install", "-y", "expecttest", "libglib", "pango", "-c", "conda-forge"])
  File "/usr/lib/python3.10/subprocess.py", line 364, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python3.10/subprocess.py", line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.10/subprocess.py", line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.10/subprocess.py", line 1845, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'conda'



---------------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/user/try/benchmark/install.py", line 114, in <module>
    raise RuntimeError("Failed to complete setup")
```

**Comment by xuzhao9** [comment]
@dany-nonstop We recently fixed this issue in https://github.com/pytorch/benchmark/pull/1528, can you update and try again?

**Comment by dany-nonstop** [comment]
> @dany-nonstop We recently fixed this issue in #1528, can you update and try again?

yes it's gone!
